src/Symfony/Component/Console/Tests/Helper/ProgressBarTest.php

```diff
@@ -535,7 +535,7 @@ public function testRedrawFrequencyIsAtLeastOneIfZeroGiven()
     public function testRedrawFrequencyIsAtLeastOneIfSmallerOneGiven()
     {
         $bar = new ProgressBar($output = $this->getOutputStream(), 0, 0);
-        $bar->setRedrawFrequency(0.9);
+        $bar->setRedrawFrequency(0);
```

Comment:
the annotation on setRedrawFrequency() should be fixed also

Reply:
Done.